suite-native/transactions/src/components/TransactionDetail/TransactionDetailWarning.tsx

```diff
@@ -0,0 +1,51 @@
+import { Box, Card, Text } from '@suite-native/atoms';
+import { Icon } from '@suite-common/icons';
+import { Translation } from '@suite-native/intl';
+import { Link } from '@suite-native/link';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+const cardStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.backgroundAlertRedSubtleOnElevation1,
+    gap: utils.spacings.small,
+    alignItems: 'center',
+
+    shadowColor: 'rgba(0, 0, 0, 0)',
+}));
+
+const titleStyle = prepareNativeStyle(utils => ({
+    gap: utils.spacings.extraSmall,
+    color: utils.colors.textAlertRed,
+    flexDirection: 'row',
+}));
+
+export const TransactionDetailWarning = () => {
+    const { applyStyle } = useNativeStyles();
+    return (
+        <Card style={applyStyle(cardStyle)}>
```

Comment:
Did you try to utilize this Atom component? `AlertBox.tsx`

Reply:
You should be able to extend the functionality a little bit and just add a new variant.